packages/manager/apps/pci-workflow/src/pages/list/List.page.tsx

```diff
@@ -0,0 +1,361 @@
+        </>
+      ),
+      label: t('pci_workflow_resource'),
+    },
+    {
+      id: 'schedule',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {workflow.cron}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_schedule'),
+    },
+    {
+      id: 'lastExecution',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {workflow.lastExecution}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_last_execution'),
+    },
+    {
+      id: 'lastExecutionStatus',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsChip
+            color={(() => {
+              switch (workflow.lastExecutionStatus) {
+                case 'SUCCESS':
+                  return ODS_THEME_COLOR_INTENT.success;
+                case 'PAUSED':
+                  return ODS_THEME_COLOR_INTENT.warning;
+                case 'ERROR':
+                  return ODS_THEME_COLOR_INTENT.error;
+                default:
+                  return ODS_THEME_COLOR_INTENT.info;
+              }
+            })()}
+            inline
+          >
+            <OsdsText
+              level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              color={ODS_THEME_COLOR_INTENT.text}
+              className="block"
+            >
+              {workflow.lastExecutionStatus}
+            </OsdsText>
+          </OsdsChip>
+        </>
+      ),
+      label: t('pci_workflow_last_execution_status'),
+    },
+    {
+      id: 'actions',
+      cell: (workflow: TWorkflow) => (
+        <div className="min-w-16">
+          <Actions projectId={projectId} backupId={workflow.id} />
+        </div>
+      ),
+      label: t(''),
+    },
+  ];
+
+  return (
+    <>
+      <HidePreloader />
+      {project && (
+        <OsdsBreadcrumb
+          items={[
+            {
+              href: projectUrl,
+              label: project.description,
+            },
+            {
+              label: t('pci_workflow_title'),
+            },
+          ]}
+        />
+      )}
+      <div className="header mb-6 mt-8">
+        <div className="flex items-center justify-between">
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._600}
+            color={ODS_THEME_COLOR_INTENT.primary}
+          >
+            {t('pci_workflow_title')}
+          </OsdsText>
+          <PciGuidesHeader category="kubernetes"></PciGuidesHeader>
+        </div>
+      </div>
```

Comment:
Lastly, we started using the https://ovh.github.io/manager/storybook-static/?path=/story/content-headers--header-with-guides from manager-component to make all apps consistent.